bin/backup.sh

```diff
@@ -28,23 +30,28 @@ for i in {1..5}; do
 	echo "Dumping database cluster $i: $PGUSER@$PGHOST:$PGPORT"
 
 	# Wait for PostgreSQL to become available.
-	COUNT=0
+	count=0
 	until psql -l > /dev/null 2>&1; do
-		if [[ "$COUNT" == 0 ]]; then
+		if [[ "$count" == 0 ]]; then
 			echo "Waiting for PostgreSQL to become available..."
 		fi
-		(( COUNT += 1 ))
+		(( count += 1 ))
+		[ $count -lt $max_pg_wait_count ] || break
```

Comment:
Why `[]` here instead of `[[]]`? Can we stick to `[[]]`, or even `if [[ ... ]]; then ... fi` (as above) for consistency.

Reply:
Mostly because I pretty much never use [[ in my own scripts - its utility is generally tiny and it is less portable. I'll tweak this for consistency though.